pkgs/data/fonts/monaspace/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "monaspace";
+  version = "1.000";
+
+  src = fetchzip {
+    url = "https://github.com/githubnext/monaspace/releases/download/v${version}/monaspace-v${version}.zip";
+    hash = "sha256-H8NOS+pVkrY9DofuJhPR2OlzkF4fMdmP2zfDBfrk83A=";
+    stripRoot = false;
+  };
+
+  outputs = [ "out" "otf" "variable" ];
```

```suggestion
  outputs = [ "out" "variable" ];
```

There is no sense on duplicate the files.

---

Done